lib/grape/locale/en.yml

```diff
@@ -10,6 +10,9 @@ en:
         values: 'does not have a valid value'
         except_values: 'has a value not allowed'
         same_as: 'is not the same as %{parameter}'
+        length_mismatch_both: 'is expected to have a length within %{min_length} and %{max_length}'
```

Comment:
potentially better wordsmithing:

is expected to be between ... and ... characters long
is expected to be at least ... characters long
is expected to be less than ... characters long

Reply:
I hesitate to use `characters long` since it doesn't make sense for arrays.